KsApi/models/Checkout.swift

```diff
@@ -2,22 +2,17 @@ import Foundation
 
 public struct Checkout: Decodable {
```

Comment:
Updated `Checkout` to receive the new properties. Currently it's shared with `CreateBacking` but that mutation is set to change so I figured we could do this for now.